calico_rkt/logutils.py

```diff
@@ -0,0 +1,17 @@
+#!/bin/python
+import logging
+import os
+
+LOG_DIR = '/var/log/calico/rkt-plugin'
+
+def configure_logger(logger, log_dir=None):
```

Comment:
Minor nit/code style point:

I'm not really convinced that this function justifies its own module; do we expect to call configure_logger from anywhere other than calico_rkt.py? And do we expect to add other logutils to this module in the future? If the answer to either is yes then this module is justified, but I think the answer to both questions is no.

It might be enough code to put in a function, but I'd just put that in calico_rkt.py. Or you could just put the 7 lines from this function that we actually use (11-17) at the top of calico_rkt.py. Again, minor nit; but worth trying to keep the project structure simple where possible (particularly in these greenfield situations where we're starting a new repo and have the chance to keep it minimally complex).